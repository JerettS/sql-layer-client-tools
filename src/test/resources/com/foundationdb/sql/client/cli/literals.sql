SELECT NULL;
SELECT 1;
SELECT 3.14;
SELECT 'asdf';
SELECT x'000102';
SELECT DATE '2013-12-16';
SELECT 1 "one", 2 "two", 3 "three";
SELECT 100, 200, 300;
