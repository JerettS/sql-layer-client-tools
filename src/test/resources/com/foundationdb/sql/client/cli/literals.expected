 _SQL_COL_1 
------------
            
(1 row)

 _SQL_COL_1 
------------
          1 
(1 row)

 _SQL_COL_1 
------------
       3.14 
(1 row)

 _SQL_COL_1 
------------
 asdf       
(1 row)

  _SQL_COL_1  
--------------
 \000\001\002 
(1 row)

 _SQL_COL_1 
------------
 2013-12-16 
(1 row)

 one | two | three 
-----+-----+-------
   1 |   2 |     3 
(1 row)

 _SQL_COL_1 | _SQL_COL_2 | _SQL_COL_3 
------------+------------+------------
        100 |        200 |        300 
(1 row)

  _SQL_COL_1   
---------------
 Hello, world! 
(1 row)

